contracts/Llama.vy

```diff
@@ -633,6 +633,14 @@ def start_al_mint():
     self.al_mint_started = True
 
 
+@external
+def stop_al_mint():
```

Comment:
Turns out we do want to stop the al mint before the auctions start so adding this.